rubix-core/src/main/java/com/qubole/rubix/core/BasicCachingFileSystemStats.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2019. Qubole Inc
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License. See accompanying LICENSE file.
+ */
+package com.qubole.rubix.core;
```

Comment:
Would `rubix.spi` be a better place to put this ? Maybe in future we can expose these stats to be consumed by user written plugins like in Presto.